doc/sections/emacs.md

```diff
@@ -66,7 +66,7 @@ Emacs, select it and use `M-;`.
 
 #### Entering unicode
 **Company:**
-if [`company-mode`](https://github.com/company-mode/company-mode) and
+if [`company`](https://github.com/company-mode/company-mode) and
```

Comment:
here I believe it's better to refer to company by it's melpa name
otherwise users not aware of emacs terminology could wrongly believe there is 3 packages to install : company company-mode and company-math